specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4143,6 +4181,24 @@
       "type": "string",
       "in": "path",
       "x-ms-skip-url-encoding": true
+    },
+    "RecognitionModel": {
```

Comment:
These two `RecognitionModel` should be camel case `recognitionModel`.

Reply:
Done.